core/src/com/mygdx/game/models/Battle.java

```diff
@@ -0,0 +1,26 @@
+package com.mygdx.game.models;
+
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+
+public class Battle {
+    Set<Integer> playerIds;
+    Map<Integer, List<BasicUnit>> units;
```

Comment:
Can a player have multiple units on the same time? Whats the point of support then?